app/controllers/dashboard_controller.rb

```diff
@@ -10,6 +11,46 @@ def index
     end
   end
 
+  def create
+    Dashboard::Manager.add_widget_to_user(User.current, params[:widget])
+    redirect_to root_path
+  end
+
+  def destroy
+    if @widget.exists? && @widget.user == User.current
```

Comment:
This line is is throwing `NoMethodError (undefined method `exists?' for #<Widget:0x007f22d55ba280>):` whenever I try to remove a widget after I restored my widgets to the defaults.